eng/build-utils.ps1

```diff
@@ -17,41 +17,41 @@ $bootstrapConfiguration = if (Test-Path variable:bootstrapConfiguration) { $boot
 $properties = if (Test-Path variable:properties) { $properties } else { @() }
 
 function GetProjectOutputBinary([string]$fileName, [string]$projectName = "", [string]$configuration = $script:configuration, [string]$tfm = "net472", [string]$rid = "", [bool]$published = $false) {
-  $projectName = if ($projectName -ne "") { $projectName } else { [System.IO.Path]::GetFileNameWithoutExtension($fileName) }
-  $publishDir = if ($published) { "publish\" } else { "" }
-  $ridDir = if ($rid -ne "") { "$rid\" } else { "" }
-  return Join-Path $ArtifactsDir "bin\$projectName\$configuration\$tfm\$ridDir$publishDir$fileName"
+ $projectName = if ($projectName -ne "") { $projectName } else { [System.IO.Path]::GetFileNameWithoutExtension($fileName) }
```

Comment:
This doesn't look right. There is only a single space indent.

Reply:
stupid off by one errors ...

Reply:
Fixed